overrides/app/pages/algolia-product-list/partials/algolia-pagination.jsx

```diff
@@ -9,79 +9,110 @@ const AlgoliaPagination = (props) => {
     const {currentRefinement, pages, refine, nbPages} = usePagination(rest)
 
     return (
-        <OrderedList
-            listStyleType="none"
-            display="flex"
-            flexShrink="0"
-            margin="0"
-            className="aa-pagination"
-        >
-            <ListItem>
-                <Link
-                    px="3"
-                    py="3"
-                    border="1px"
-                    borderLeftRadius="2xl"
-                    borderColor="gray.100"
-                    _hover={{
-                        backgroundColor: 'gray.50'
-                    }}
-                    onClick={(event) => {
-                        event.preventDefault()
-                        currentRefinement != 0 && refine(currentRefinement - 1)
+        <>
+            <Flex align="center" className="wt-pagination-mobile">
+                <Box>Page</Box>
+                <Select
+                    value={currentRefinement}
```

Comment:
Why are we building a custom component instead of using the builtin `Pagination`? https://www.algolia.com/doc/api-reference/widgets/pagination/react/
Are there missing features?

Reply:
I can convert it to our pagination but I couldn't make it so responsive when there are many pages..

Reply:
Can you show what was happening? If it's acceptable, I'd prefer to use our components and fix issues in them directly rather than having to re-write them.

Reply:
I found a way for it, after I removed `showNext` buttons it also worked for responsive,
I was thinking it is listing all the pages but only put 7 pages and fixed the style

Reply:
Great! Thanks.